api/src/authentication/application/token.route.js

```diff
@@ -0,0 +1,39 @@
+import Joi from 'joi';
+
+import { securityPreHandlers } from '../../shared/application/security-pre-handlers.js';
+import { tokenController } from './token-controller.js';
+
+export const tokenRoutes = [
```

Comment:
j'aurais bien fusionné le commit `feat(api): add token route module` dans le commit `refactor(api): move saml routes from lib to src`